### DISCUSSION TITLE
Downloads have doubled...

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#635] [General]
Just a thought.  When you unleash 1.3, you might want to change this since downloads have doubled:
![image](https://user-images.githubusercontent.com/67484051/164558377-91d2b74b-b924-4f55-849b-6025fbcfd205.png)

**Comment by worksofliam** [comment]
Very good point. Feel free to PR or I will sort it very soon!

**Reply by SJLennon** [reply]
PR created.  Forgot I could do things like this...
